### ISSUE TITLE
block every new device by default option.

### ISSUE THREAD

**Issue by rustdesk** [rustdesk/rustdesk-server-pro#166]
> I would like to ask how to secure the server so that no one can use mine. 
I mean if someone finds out my public key they can connect to my server and use it. 
Is there any way to limit this? or is it possible to block every new device, so when someone connects I have to manually allow them through the RustDesk Console?

**Comment by pdifeo** [comment]
Perhaps I have said it before, an email alert to the administrator might also be useful.

**Comment by rustdesk** [comment]
> Perhaps I have said it before, an email alert to the administrator might also be useful.

Yes, this will be next step. @21pages Let's do this later, please continue your other jobs.

For email notification, we need to think about how to avoid too frequment mails. e.g. 300 new devices in 1 minute,  we can not send 300 mails.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server-pro/releases/tag/1.1.11

![image](https://github.com/rustdesk/rustdesk-server-pro/assets/71636191/5cd67663-1b1e-4172-a05d-116d8a5fdce2)
